.editorconfig

```diff
@@ -164,11 +164,11 @@ dotnet_naming_rule.interface_should_be_begins_with_i.style = begins_with_i
 
 dotnet_naming_rule.types_should_be_pascal_case.severity = suggestion
 dotnet_naming_rule.types_should_be_pascal_case.symbols = types
-dotnet_naming_rule.types_should_be_pascal_case.style = pascal_case
+dotnet_naming_rule.types_should_be_pascal_case.style = non_private_static_field_style
```

Comment:
Opening the editorconfig made a bunch of changes, and we changed one thing for protected internal fields to format the way we expect.

If you see any IDE recommendations for style that don't match what we expect, let me know.